site/en/docs/privacy-sandbox/feedback/report-2023-q2.md

```diff
@@ -5,7 +5,7 @@ subhead: >
   Quarterly report for 2023 Q2 summarising the ecosystem feedback received on
   Privacy Sandbox proposals and Chrome's response.
 description: >
-  Quarterly report for 2023 Q1 summarising the ecosystem feedback received on
+  Quarterly report for 2023 Q2 summarising the ecosystem feedback received on
```

Comment:
"summarizing" (for US English)?

Reply:
I've always left that because it's CMA. Should I change?

Reply:
it's been that way for a few years

Reply:
Ah — sorry. Forgot that this is CMA language.

Reply:
well, now I don't know :). I just changed them all :)